index.yaml

```diff
@@ -12,8 +12,8 @@ indexes:
 
 - kind: AppFeedbackReportModel
   properties:
-  - name: created_on
   - name: scrubbed_by
+  - name: created_on
```

Comment:
Hmmm, does the order actually matter here?

Reply:
Not sure but we decided to try this per https://github.com/oppia/oppia/issues/15194#issuecomment-1081718715

Reply:
Hmm, okay, yeah I think it is worth testing, if this works we should make sure that the scripts that we have for generating _index.yaml_ take that into account.

Reply:
It does matter. See https://cloud.google.com/appengine/docs/legacy/standard/python/config/indexref#index_definitions -- there is a specific order in which index properties need to be listed.

Reply:
Thanks @seanlip for confirming. @vojtechjelinek how do we generate the yaml file? I couldn't figure out how to do it, so I edited this manually (which is non ideal). I can update the logic in the script as part of this PR if required.

Reply:
We use https://github.com/oppia/oppia/blob/develop/scripts/extend_index_yaml.py

Reply:
Thanks @vojtechjelinek! I think this can be done in a separate. Created https://github.com/oppia/oppia/issues/16010 to track.

Reply:
Sure!